const enum does not work when isolatedModules is enabled

When using this lib (transitive via cherrio -> domhandler -> domelementtype) tsc complains with

```
error TS2748: Cannot access ambient const enums when the '--isolatedModules' flag is provided.
```

when running with `--isolatedModules`. Source code:

https://github.com/fb55/domelementtype/blob/71814099d360d9ced25fadcadd44acf22d827455/src/index.ts#L38

You can find more infos about this in the popular lib chalk and how they solved this: https://github.com/chalk/chalk/issues/363

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,5 +1,5 @@
 /** Types of elements found in htmlparser2's DOM */
-export const enum ElementType {
+export enum ElementType {
     /** Type for the root element of a document */
     Root = "root",
     /** Type for Text */

